calculator.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="../week4/calculator.css">
+    <title> Calculator </title>
+  </head>
+  <body>
+        <ul class="numList">
+        </ul>
+        <div class="inputTray">
+            <input type="text" class="inputYourNumber"/> <br><br>
+            <button type="button" class="addNumber">Add a number</button>
+        </div>
+        Sum: <p class="sum">0</p>
+    
+    <script>
+    const inputYourNumberEl = document.querySelector('.inputYourNumber');
+    const addBtnEl = document.querySelector('.addNumber');
+    const sumEl = document.querySelector('.sum');
+    const numListEl = document.querySelector('.numList');
+    let sum = 0;
+
+    function handleAddBtnClick() {
+        const numberToAdd = parseFloat(inputYourNumberEl.value);
+
+        sum = sum + numberToAdd; // update sum
+        sumEl.textContent = sum; // update sum element
+
+        const newNumberEl = document.createElement('li'); // <li></li>
+        newNumberEl.textContent = numberToAdd;
+        numberListEl.appendChild(numberEl);
```

Comment:
Wow nice work almost got it! The problem is with this link 32.  The `numberListEl` variable does not exist did you mean `numListEl` also the variable `numberEl` does not exist did you mean `newNumberEl`?

I got your code to work with the following changes:

```diff
         const newNumberEl = document.createElement('li'); // <li></li>
         newNumberEl.textContent = numberToAdd;
-        numberListEl.appendChild(numberEl);
+        numListEl.appendChild(newNumberEl);
         inputYourNumberEl.value = '';
```